src/internal/hooks/useMessage.ts

```diff
@@ -216,3 +226,67 @@ const statusSetter = (setStatus: Dispatch<SetStateAction<MessageStatus>>) => {
     }
   };
 };
+
+export const useMessageSenders = () => {
+  const { sendToParent: iFrameLoaded } = useSendMessageToParent(
+    "iFrameLoaded",
+    TARGET_ORIGINS
+  );
+
+  const { sendToParent: sendDevLayoutSaveStateData } = useSendMessageToParent(
+    "sendDevSaveStateData",
+    TARGET_ORIGINS
+  );
+
+  const { sendToParent: pushPageSets } = useSendMessageToParent(
+    "pushPageSets",
+    TARGET_ORIGINS
+  );
+
+  const { sendToParent: saveLayoutSaveState } = useSendMessageToParent(
+    "saveSaveState",
+    TARGET_ORIGINS
+  );
+
+  const { sendToParent: deleteLayoutSaveState } = useSendMessageToParent(
+    "deleteSaveState",
+    TARGET_ORIGINS
+  );
+
+  const { sendToParent: publishVisualConfiguration } = useSendMessageToParent(
+    "saveVisualConfigData",
+    TARGET_ORIGINS
+  );
+
+  const { sendToParent: saveThemeSaveState } = useSendMessageToParent(
+    "saveThemeSaveState",
+    TARGET_ORIGINS
+  );
+
+  const { sendToParent: deleteThemeSaveState } = useSendMessageToParent(
+    "deleteThemeSaveState",
+    TARGET_ORIGINS
+  );
+
+  const { sendToParent: publishThemeConfiguration } = useSendMessageToParent(
```

Comment:
Should we group things that are common vs layout vs theme? I think that would be nice. Common would be the things anything would need. The other two are for their own specifics.